src/components/feedback/Feedback.js

```diff
@@ -0,0 +1,211 @@
+import React, { Component } from 'react'
+import { View, Text, StyleSheet, TextInput, Button, ScrollView } from 'react-native'
+import Icon from 'react-native-vector-icons/FontAwesome'
+import style from '../../common/style'
+import Toast, { DURATION } from 'react-native-easy-toast'
+
+import { FeedbackCriteria } from './FeedbackCriteria'
+import * as feedbackAction from '../../actions/feedback'
+
+import { connect } from 'react-redux'
+
+class Feedback extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      titleText: "Rate this session",
+      categories: [
+        { id: 'Overall', title: 'Session rating', low: 'Not so awesome', high: 'Awsome' }, 
+        { id: 'Relevance', title: 'How relevant was this session to your projects', low: 'Not at all', high: 'Extremly' }, 
+        { id: 'Content', title: 'Based on the decription/my expectaions, the content was', low: 'Too basic', high: 'Too advanced' },
+        { id: 'Quality', title: 'Speaker quality', low: 'Poor', high: 'Outstanding' }, 
+        { id: 'Comment', title: 'Any other comments' }
+      ],
+      feedbackData: this.props.feedbackData
```

Comment:
I'm trying to keep the views stateless and by doing so I find them easier to understand and test the components. The initial state here isn't needed imo since most of it is static data (`categories`, `titleText`) and the other ones comes from the `props`. If we need to store something then we can use the redux store.  By doing so we're following the flux pattern that for redux help us implement (aka unidirectional data flow).